### PR TITLE
LPD-5877 Functional test for COMMERCE-11835

### PR DESCRIPTION
**Pull request by joanagrego**
https://liferay.atlassian.net/browse/LPD-5877

### CODE REVIEW

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/test/playwright/tests/commerce/commerceDiagramProduct.spec.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		await commerceAdminProductPage.gotoProduct(product.name['en_US']);
+
+		await commerceAdminProductDetailsPage.goToProductDiagram();
+
+		await commerceAdminProductDetailsDiagramPage.goToDragAndDropImages();
+
+		await expect(
+			commerceAdminProductDetailsDiagramPage.dragAndDropImages
+		).toBeVisible({
+			timeout: 2000,
```
Comment (smottal) [inline modules/test/playwright/tests/commerce/commerceDiagramProduct.spec.ts:69]: is this needed?

**Review by smottal — COMMENTED** [review]
@joanagrego just a question

File: modules/test/playwright/tests/commerce/commerceDiagramProduct.spec.ts (hunk 1 above)
Comment (joanagrego) [reply modules/test/playwright/tests/commerce/commerceDiagramProduct.spec.ts:69]: This component appears after a loading section. I tried setting a smaller timeout but it broke on ci. Do we have another approach to work with the component visibility after a load time? @smottal

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by kevenleone** [comment]
resent: https://github.com/is-solutions-delivery/liferay-portal/pull/5596

**Pull request closed without merging**